src/main/java/frc/robot/subsystems/SuperStructure.java

```diff
@@ -83,12 +84,36 @@ public Command coralLevelFour(BooleanSupplier score) {
                         .until(armPivot.atAngle(ArmPivot.CORAL_POST_SCORE))),
                 score),
             Commands.print("Pre preIntake()"),
-            coralPreIntake(),
+            coralPreIntake().unless(RobotModeTriggers.autonomous()),
             Commands.print("Post preIntake()"))
         .deadlineFor(colorSet(0, 255, 0, "Green - Aligned With L4").asProxy())
         .withName("Coral Level 4");
   }
 
+
+  public Command coralLevelFourAuto(BooleanSupplier score) {
+    return Commands.sequence(
+            Commands.parallel(
+                    Commands.print("Pre position"),
+                    elevator
+                        .setLevel(ElevatorSubsystem.CORAL_LEVEL_FOUR_PRE_POS)
+                        .deadlineFor(
+                            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_UP).until(score)),
+                    spinnyClaw.stop())
+                .withTimeout(0.7),
+            repeatPrescoreScoreSwing(
+                Commands.sequence(
+                    Commands.parallel(
+                            elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_FOUR_PRE_POS),
+                            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_PRE_L4))
+                        .withDeadline(Commands.waitUntil(score)),
+                    armPivot
+                        .moveToPosition(ArmPivot.CORAL_PRESET_DOWN)
+                        .withTimeout(1.5)
+                        .until(armPivot.atAngle(ArmPivot.CORAL_POST_SCORE))),
+                score));
+  }
+
```

Comment:
Could you delete this please, now that we're using the normal `coralLevelFour()` command factory method?